*** mdrun.c	2011-11-19 00:59:53.000000000 +0100
--- /Users/carlo/Downloads/gromacs-4.5.5/src/kernel/mdrun.c	2011-09-21 14:34:32.000000000 +0200
***************
*** 52,58 ****
  #ifdef GMX_THREADS
  #include "thread_mpi.h"
  #endif
- #include "solute.h"
  
  /* afm stuf */
  #include "pull.h"
--- 52,57 ----
***************
*** 420,427 ****
    output_env_t oenv=NULL;
    const char *deviceOptions = "";
  
-   rex.t0=-1.;
- 
    t_pargs pa[] = {
  
      { "-pd",      FALSE, etBOOL,{&bPartDec},
--- 419,424 ----
***************
*** 478,485 ****
        "Do multiple simulations in parallel" },
      { "-replex",  FALSE, etINT, {&repl_ex_nst}, 
        "Attempt replica exchange every # steps" },
-     { "-sol_temp",  FALSE, etREAL, {&rex.t0}, 
-       "Reference temperature for Solute Tempering" },
      { "-reseed",  FALSE, etINT, {&repl_ex_seed}, 
        "Seed for replica exchange, -1 is generate a seed" },
      { "-rerunvsite", FALSE, etBOOL, {&bRerunVSite},
--- 475,480 ----
***************
*** 600,606 ****
        {
            sim_part = sim_part_fn + 1;
        }
!       if (MULTISIM(cr) && MASTER(cr))
        {
            check_multi_int(stdout,cr->ms,sim_part,"simulation part");
        }
--- 595,602 ----
        {
            sim_part = sim_part_fn + 1;
        }
! 
!       if (MULTISIM(cr))
        {
            check_multi_int(stdout,cr->ms,sim_part,"simulation part");
        }
